client/src/components/pages/Home/index.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import Button from '../../common/Button';
+import './index.css';
+
+const HomePage = () => {
+  return (
+    <div className="home">
+      <div className="home__header">
+        <div className="home__header__img__overlay"></div>
+      </div>
+      <div className="about">
+        <h3 className="about__title">About</h3>
+        <div className="about__block">
```

Comment:
seems `about__block` is redundant, there are no styles for it and hence it does not add anything, pls remove